elastic/beats
# Enable x-pack security for testing against snapshots

## What does this PR do?

Refs https://github.com/elastic/beats/issues/27211

This turns on xpack security for testing against snapshots (currently against 8.0.0). 

## Why is it important?

Per the [requesting issue](https://github.com/elastic/beats/issues/27211): `As Elasticsearch is enabling security by default in 8.0 we should also do this for our testing to uncover potential issues early on.`

## Checklist




## Author's Checklist



## How to test this PR locally

Run `make up` from `testing/environments`. Ensure that the environment comes up successfully. 

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes https://github.com/elastic/beats/issues/27211
